src/Table/TableCell.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import css from 'styled-jsx/css'
+
+import { Consumer } from './tableContext'
+
+const tableCellStyles = css`
+    td {
+        border-bottom: 1px solid #e8edf2;
+        padding: 0 12px;
+        font-size: 14px;
+    }
+
+    div {
+        min-height: 45px;
+    }
+
+    :global(tfooter) div {
+        min-height: 36px;
+    }
+`
+
+const TableCellStatic = ({ children, colSpan, rowSpan }) => (
+    <td colSpan={colSpan} rowSpan={rowSpan}>
+        <div>{children}</div>
+        <style jsx>{tableCellStyles}</style>
+    </td>
+)
+
+const TableCellResponsive = ({ children, colSpan, rowSpan, title }) => (
+    <td colSpan={colSpan} rowSpan={rowSpan}>
+        <div>{children}</div>
+        <style jsx>{tableCellStyles}</style>
+        <style jsx>{`
+            @media (max-width: 768px) {
+                td {
+                    display: table-row;
+                    width: 100%;
+                }
+
+                td:before {
+                    content: '${title}:';
+                    display: table-cell;
+                    white-space: nowrap;
+                    padding: 0 16px;
+                    font-weight: bold;
+                }
+
+                :global(tfoot) td:before {
+                    display: none;
+                }
+
+                div {
+                    display: table-cell;
+                    width: 100%;
+                    padding: 0 16px;
+                }
+            }
+
+            @media (max-width: 400px) {
+                td {
+                    display: block;
+                }
+
+                td:first-child {
+                    margin-top: 0;
+                }
+
+                td:before {
+                    display: block;
+                    white-space: normal;
+                    min-height: 24px;
+                    line-height: 18px;
+                    padding: 8px 0 0 0;
+                }
+
+                div {
+                    display: block;
+                    padding: 0;
+                    min-height:32px;
+                }
+            }
+        `}</style>
+    </td>
+)
+
+export const TableCell = ({ children, title, colSpan, rowSpan }) => (
```

Comment:
I understand why we have a `title` property on a cell, but it makes for pretty verbose and unwieldy usage.  To make a simple 4x4 table with a header I need to duplicate 4x (and maintain those duplicates) of all 4 column names.  That's 16 redundant values, and it's much worse for more complicated situations.

Reply:
If you want a simple table, you can just use the table with `staticLayout`, then you don't need that one anymore

Reply:
Yeah but then it's static and not responsive...

Reply:
You could accomplish this with `:nth-child` CSS rules on the `Table` component, set on the `Table` context by the order of the `Head` components

Reply:
(though that introduces some implicit dependence, it could also be accomplished independently on the `TableCell` component with some context voodoo)